wiki/osu!academy/id.md

```diff
@@ -1,63 +1,57 @@
-| [![](img/Academy_28.png)](https://www.youtube.com/watch?v=PEZFOM8NKtw "Episode 28")   | Pada episode ini, kami akan mengemas perihal dasar-dasar forum, misalnya dimana anda dapat memposting dan membenarkan kelakuan anda!                                                   |
-| [![](img/Academy_29.png)](https://www.youtube.com/watch?v=uvCRwcyJ4TA "Episode 29")   | Pada episode ini, kami akan berbincang mengenai dasar-dasar pembuatan storyboard!                                                                                                      |
-| [![](img/Academy_30.png)](https://www.youtube.com/watch?v=EvICgPuOylk "Episode 30")   | Pada episode ini, kami akan mengemas mengenai pembuatan storyboard yang lebih canggih!                                                                                                 |
-| [![](img/Academy_31.png)](https://www.youtube.com/watch?v=s2ZK4o8V5tI "Episode 31")   | Pada episode ini, kami akan lebih mengacu lebih dalam mengenai *guest difficulty* dan kontribusi-kontribusi beatmap lainnya!                                                           |
-| [![](img/Academy_32.png)](https://www.youtube.com/watch?v=wa_hNegtBw0 "Episode 32")   | Pada episode ini, kami akan berbincang mengenai sistem peringkat yang baru!                                                                                                            |
-| [![](img/Academy_33.png)](https://www.youtube.com/watch?v=_-Xke2bqzok "Episode 33")   | Pada episode ini, kami akan mengemas hal-hal mengenai tips editor yang lebih canggih!                                                                                                  |
-
-
-
-Klik pada gambar untuk menyaksikan episode yang diinginkan.
+# osu!academy
+
+![Logo osu!academy](img/osu!academy-logo.jpg "Logo osu!academy")
+
+**osu!academy** memiliki tujuan untuk memberikan pandangan lengkap terhadap semua aspek osu!, mulai dari bermain hingga mapping dan segala sesuatu yang ada di dalamnya.
+
+Kami berharap pemain baru dapat menemukan video osu!academy ini informatif, dan akan membantu meringankan hal yang harus dipelajari dari game ini. 
+Jika Anda memiliki pertanyaan atau komentar, Anda bisa mengirimnya kepada [ztrot](/users/6347), dia adalah pemilik satu-satunya osu!academy bersamaan dengan kanal YouTube-nya.
+
+[Berikut merupakan trailernya](https://www.youtube.com/watch?v=z5gy34k3RI0&feature=c4-overview&list=UUMeRgqzTfC5ja40B6kM6pdg).
+
+## Daftar Episode
+
+| Thumbnail Episode | Tautan | Deskripsi |
+|----------------------------------------------------------|--------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ![](img/Academy_1.png) | [Episode 1](https://www.youtube.com/watch?v=cz522ZAs5aQ "Episode 1") | Dalam episode ini, kita akan membahas cara penginstalan dan pembuatan akun! |
```

Comment:
`membahas cara penginstalan dan pembuatan akun!` will make users think `how to install account ?`

> membahas cara memasang osu! dan membuat sebuah akun

will make something clearly